Makefile

```diff
@@ -0,0 +1,15 @@
+TAGS="sqlite vbuffalo"
```

Comment:
@stanislas-m you're stretching my Makefile skills here. :)

Reply:
From https://ftp.gnu.org/old-gnu/Manuals/make-3.79.1/html_chapter/make_6.html:

```
There is another assignment operator for variables, `?='. This is called a conditional variable assignment operator, because it only has an effect if the variable is not yet defined. This statement:

FOO ?= bar

is exactly equivalent to this (see section The origin Function):

ifeq ($(origin FOO), undefined)
  FOO = bar
endif

Note that a variable set to an empty value is still defined, so `?=' will not set that variable. 
```